docs/config/README.md

```diff
@@ -624,15 +625,21 @@ The module will be shown only if any of the following conditions are met:
 
 ### Options
 
-| Variable   | Default          | Description                                                                  |
-| ---------- | ---------------- | ---------------------------------------------------------------------------- |
-| `symbol`   |                  | The symbol used before displaying the variable value.                        |
-| `variable` |                  | The environment variable to be displayed.                                    |
-| `default`  |                  | The default value to be displayed when the selected variable is not defined. |
-| `prefix`   | `""`             | Prefix to display immediately before the variable value.                     |
-| `suffix`   | `""`             | Suffix to display immediately after the variable value.                      |
-| `style`    | `"dimmed black"` | The style for the module.                                                    |
-| `disabled` | `false`          | Disables the `env_var` module.                                               |
+| Option     | Default                                | Description                                                                  |
+| ---------- | -------------------------------------- | ---------------------------------------------------------------------------- |
+| `symbol`   |                                        | The symbol used before displaying the variable value.                        |
+| `variable` |                                        | The environment variable to be displayed.                                    |
+| `default`  |                                        | The default value to be displayed when the selected variable is not defined. |
+| `format`   | `"with [${env_value}]($style) "` | The format for the module.                                                   |
+| `disabled` | `false`                                | Disables the `env_var` module.                                               |
+
+### Variables
+
+| Variable  | Example             | Description                             |
+| --------- | ------------------- | --------------------------------------- |
+| env_value | `Windows NT`        | The environment value of $OS on Windows |
```

Comment:
`env_value` isn't specific about `$OS`. This could be explained in a more generic manner.

```
The environment value of option `variable`
```

Reply:
Oops. I tested locally with the `$OS` variable and my mind tried to explain what the example was about.